examples/aws-cloudtrail/main.tf

```diff
@@ -29,6 +29,42 @@ locals {
     module     = "quantum-sec/package-aws-security/examples/aws-cloudtrail"
   }
   tags = merge(local.default_tags, var.tags)
+
+  aws_region     = var.aws_region
+  aws_account_id = var.aws_account_id
```

Comment:
LGTM, just a small nitpick on an inconsistency regarding how `aws_region` and `aws_account_id` were used.
https://github.com/quantum-sec/package-aws-security/blob/356a01e74ed347e2aef39f36dd2fe5d1714f4b0f/examples/aws-cloudtrail/main.tf#L33-L34
https://github.com/quantum-sec/package-aws-security/blob/356a01e74ed347e2aef39f36dd2fe5d1714f4b0f/examples/aws-cloudtrail/main.tf#L85-L86